mmengine/runner/runner.py

```diff
@@ -367,8 +367,9 @@ def __init__(
         mmengine.mkdir_or_exist(self._log_dir)
         # Used to reset registries location. See :meth:`Registry.build` for
         # more details.
-        self.default_scope = DefaultScope.get_instance(
-            self._experiment_name, scope_name=default_scope)
+        if default_scope is not None:
+            self.default_scope = DefaultScope.get_instance(
+                self._experiment_name, scope_name=default_scope)
```

Comment:
If `default_scope` is None, the attribute `default_scope` will not be set. If users access this attribute, an exception will be thrown.